scripts/bootstrap

```diff
@@ -0,0 +1,200 @@
+        WGET_BINARIES_FILENAME='wget-remarkable-pipeline_job245_wget1.20.3.zip'
+        WGET_BINARIES_SHA256='84185a5934e34e25794d439c78dc9f1590e4df12fbf369236f6a8749bf14d67f'
+
+        # Download and compare to hash
+        wget "$WGET_BINARIES_PATH/$WGET_BINARIES_FILENAME" -O "/home/root/$WGET_BINARIES_FILENAME"
+        if ! echo "$WGET_BINARIES_SHA256  /home/root/$WGET_BINARIES_FILENAME" | sha256sum -c -
+        then
+            echo "FATAL: Invalid hash" >&2
+            exit 1
+        fi
+
+        # Ensure to /home/root/.cache/wget_bin exists and is empty
+        if [ -d /home/root/.cache/wget_bin ]
+        then
+            rm -rf /home/root/.cache/wget_bin/*
+        else
+            mkdir -p /home/root/.cache/wget_bin
+        fi
+
+        # Unzip to /home/root/.cache/wget_bin and remove downloaded file
+        unzip "/home/root/$WGET_BINARIES_FILENAME" -d /home/root/.cache/wget_bin -q
+        rm "/home/root/$WGET_BINARIES_FILENAME"
+
+        cat > /home/root/.cache/wget_bin/wget <<EOF
+#!/bin/sh
+LD_LIBRARY_PATH="/home/root/.cache/wget_bin/dist" /home/root/.cache/wget_bin/dist/wget \$@
+EOF
+
+        chmod +x /home/root/.cache/wget_bin/wget
+    fi
+
+    # Ensure this binary is used
+    if [[ $(command -v wget) != /home/root/.cache/wget_bin/wget ]]; then
+        PATH="/home/root/.cache/wget_bin:$PATH"
+    fi
+}
+
+main() {
+    # Allow changing the default branch
+    [[ -n "$TOLTEC_BRANCH" ]] || TOLTEC_BRANCH="stable"
+
+    # Select source for remarkable_entware repo
+    if [ -z "$REMARKABLE_ENTWARE_REPO_AUTHOR" ]; then
+        # Use LinusCDE's fork for the time beeing
+        # TODO: Switch to a more official repo when it exists
+        REMARKABLE_ENTWARE_REPO_AUTHOR=LinusCDE
+    fi
+
+    # Check status of installation and ensure installed
+    if [ -d /home/root/.entware ]; then
+        if [[ -d /opt ]] && files="$(ls -qA -- /opt)" && [[ -n $files ]]; then
+            # Entware is installed and active
+            log INFO "Entware is installed and active"
+        else
+            # Installed but not active
+            log INFO "Entware is installed but not reenabled"
+            entware_reenable
+        fi
+    else # = not found: /home/root/.entware
+        # Not installed at all
+        log INFO "Entware is not installed"
+        entware_install
+    fi
+
+    # Ensure opkg is in PATH
+    ensure_opkg_available
+    # Ensure wget is installed as opkg package
+    ensure_opt_wget_and_ca_certs_installed
+    # Add toltec repo if mising
+    ensure_toltec_repo_added
+
+    if [ $# -ge 1 ]; then
+        install_packages "$@" || throw "Failed to install requested packages"
+    fi
+}
+
+entware_install() {
+    bootstrap_wget || throw "bootstrap_wget() failed!"
+    (wget -O- "https://raw.githubusercontent.com/$REMARKABLE_ENTWARE_REPO_AUTHOR/remarkable_entware/master/entware_install.sh" | sh) \
```

Comment:
Couldn’t we merge the `entware_install.sh` script into this script?

Reply:
Probably. I didn't feel comfortable to make that decision here though. We should talk with @evildo again to maybe get his repo either into [toltec-dev](https://github.com/toltec-dev/) (and him, too) or merge the script outright.

Reply:
I think a good solution would be to have a generic script which installs Opkg and accepts as arguments a list of repositories to enable. Then it could be used both to install Toltec and to install Entware only.

Reply:
If I'm not mistaken, remarkable_entware basicially runs a provided installer from entware (which installs it and automatically adds entware) and fixes some libs as well for the remarkable.  
So we would basicially need a combination of this one, entware_install.sh and entwares installer.